plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -200,7 +200,11 @@ private void removeSnapshotFromRegistry(final DockerMachineSource dockerMachineS
                 conn.disconnect();
             }
         } catch (IOException e) {
-            LOG.error(e.getLocalizedMessage(), e);
+            LOG.error("Failed to remove {} snapshot from {}. Cause: {}",
+                      dockerMachineSource.getRepository(),
+                      dockerMachineSource.getRegistry(),
+                      e.getLocalizedMessage(),
+                      e);
```

Comment:
You have 3 "{}",  but arguments 4. Maybe delete fourth argument if it is not used?